airbyte-integrations/connectors/source-mysql-strict-encrypt/Dockerfile

```diff
@@ -16,5 +16,5 @@ ENV APPLICATION source-mysql-strict-encrypt
 
 COPY --from=build /airbyte /airbyte
 
-LABEL io.airbyte.version=0.5.10
+LABEL io.airbyte.version=0.5.15
```

Comment:
@jdpgrailsdev is it expected that we are bumping this up by 5 versions? Is this to keep it consistent with the `source-mysql` connector?

Reply:
@lmossman I assumed that they just got out of sync because the non-strict version has been released a few times and people forgot to bump this.  It would make sense to keep them in sync, as cloud just uses the strict ones as a replacement for the non-strict version.